ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main.yaml

```diff
@@ -0,0 +1,26 @@
+base_images:
+  base:
+    name: ocp-qe-perfscale-ci
+    namespace: ci
+    tag: latest
+build_root:
+  image_stream_tag:
+    name: ci-tools-build-root
+    namespace: ci
+    tag: latest
+images:
+- dockerfile_path: prow/Dockerfile
+  from: base
+  to: ocp-qe-perfscale-ci
```

Comment:
base image and to image with the same name does not looks correct to me.

Reply:
Do I always need build_root and base_image? it is not clear to me from the docs. https://docs.ci.openshift.org/docs/architecture/ci-operator/

Reply:
I think base_image not required. Build_root is.

Reply:
Yes, you are right. base_image is not required, and build_root is required if you would like to build a image there.